### PR TITLE
Add Resnet Inference Example

### PR DESCRIPTION
**Pull request by michaelzhiluo**

### CODE REVIEW

File: examples/resnet_inference_app.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: resnet-inference
+
+resources:
+  cloud: aws
+  accelerators:
+    V100: 1
+
+inputs: {
```
Comment (romilbhardwaj) [inline examples/resnet_inference_app.yaml:8]: If we are forcing the cloud to be `aws`, can we remove the `inputs` and `outputs` field? The optimizer doesn't any options other than to use AWS in this YAML so these fields have no effect. If we want to keep them, we should remove `cloud: aws`

**Review by romilbhardwaj — COMMENTED** [review]
Thanks for adding this! left some comments

File: examples/resnet_inference_app.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+
+outputs: {
+  resnet-model-dir: 0.1,
+}
+
+file_mounts:
+  /tmp/resnet-model-dir: 
+    source: s3://mluo-resnet-model-dir
```
[the remark this answers is not in the document]
Comment (michaelzhiluo) [reply examples/resnet_inference_app.yaml:19]: Sounds good lol, made the bucket public; just retry the yaml when you have time

File: examples/resnet_inference_app.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+name: resnet-inference
```
[the remark this answers is not in the document]
Comment (michaelzhiluo) [reply examples/resnet_inference_app.yaml:1]: Done, thanks!

File: examples/resnet_inference_app.yaml (hunk 1 above)
Comment (michaelzhiluo) [reply examples/resnet_inference_app.yaml:8]: Ah it does not need to be AWS (removed the field). And remove the inputs and outputs field.

**Comment by michaelzhiluo** [comment]
@romilbhardwaj Thanks for the review, ready for another round. PTAL

File: examples/resnet_inference_app.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (romilbhardwaj) [reply examples/resnet_inference_app.yaml:15]: Still facing the same error :(
```
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples$ sky launch -c inf resnet_inference_app.yaml 
Task from YAML spec: resnet_inference_app.yaml
E 10-13 17:41:19 storage.py:592] Sky could not get StoreType.S3 store with name mluo-resnet-model-dir.
botocore.exceptions.ClientError: An error occurred (403) when calling the HeadBucket operation: Forbidden

The above exception was the direct cause of the following exception:                                  
                                                                                                      
sky.exceptions.StorageBucketGetError: Failed to connect to an existing bucket 'mluo-resnet-model-dir'.
Please check if:
  1) the bucket name is taken and/or
  2) the bucket permissions are not setup correctly. Consider using aws s3 ls mluo-resnet-model-dir to debug.

```

[the remark this answers is not in the document]
Comment (michaelzhiluo) [reply examples/resnet_inference_app.yaml:15]: Try it one more time! Think it should work now...

**Review by romilbhardwaj — APPROVED** [review]
Works now! Thanks for adding!

**Pull request merged**